Sources/SwifterSwift/HealthKit/HKActivitySummaryExtensions.swift

```diff
@@ -0,0 +1,33 @@
+//
+//  HKActivitySummaryExtensions.swift
+//  SwifterSwift
+//
+//  Created by Gilbert Lo on 7/3/20.
+//  Copyright © 2020 SwifterSwift
+//
+
+#if canImport(HealthKit)
+import HealthKit
+
+// MARK: - Properties
+@available(watchOS 2.2, *)
+public extension HKActivitySummary {
+
+    /// SwifterSwift: Check if stand goal is met
+    var isStandGoalMet: Bool {
+        return appleStandHoursGoal.compare(appleStandHours) != .orderedDescending
+    }
+
+    /// SwifterSwift: Check if exercise time goal is met
+    var isExerciseGoalMet: Bool {
```

Comment:
I think the names should match more exactly.
```suggestion
    var isExerciseTimeGoalMet: Bool {
```

Reply:
updated